Library/QuickPulseStateManager.ts

```diff
@@ -145,24 +148,33 @@ class QuickPulseStateManager {
     }
 
     private _ping(envelope: Contracts.EnvelopeQuickPulse): void {
-        this._sender.ping(envelope, this._quickPulseDone.bind(this));
+        this._sender.ping(envelope, this._rediectedHost, this._quickPulseDone.bind(this));
     }
 
     private _post(envelope: Contracts.EnvelopeQuickPulse): void {
-        this._sender.post(envelope, this._quickPulseDone.bind(this));
+        this._sender.post(envelope, this._rediectedHost, this._quickPulseDone.bind(this));
     }
 
     /**
      * Change the current QPS send state. (shouldPOST == undefined) --> error, but do not change the state yet.
      */
-    private _quickPulseDone(shouldPOST?: boolean, res?: http.IncomingMessage): void {
+    private _quickPulseDone(shouldPOST?: boolean, res?: http.IncomingMessage, 
+            redirectedHost?: string, pollingIntervalHint?: number): void {
         if (shouldPOST != undefined) {
             if (this._isCollectingData !== shouldPOST) {
                 Logging.info("Live Metrics sending data", shouldPOST);
                 this.enableCollectors(shouldPOST);
             }
             this._isCollectingData = shouldPOST;
 
+            if (redirectedHost && redirectedHost.length > 0) {
```

Comment:
Please do a console.info here to aid debugging.

Reply:
Sure. Done!